inc/agent.class.php

```diff
@@ -1485,42 +1482,19 @@ public static function getTopicsToCleanup() {
 
    /**
     * Send an geolocation request to the agent
-    * @param $errorMessage
+    *
     * @return bool
+    * @throws Exception
     */
-   protected function sendGeolocationQuery(&$errorMessage) {
-      $topic = $this->getTopic();
-      if ($topic === null) {
-         $errorMessage = __('Timeout requesting position', 'flyvemdm');
-         return false;
-      }
+   private function sendGeolocationQuery() {
 
       $computerId = $this->fields['computers_id'];
       $geolocation = new PluginFlyvemdmGeolocation();
       $lastPositionRows = $geolocation->find("`computers_id`='$computerId'", '`date` DESC, `id` DESC', '1');
       $lastPosition = array_pop($lastPositionRows);
 
-      $mqttMessage = ['query' => 'Geolocate'];
-      $this->notify("$topic/Command/Geolocate", json_encode($mqttMessage, JSON_UNESCAPED_SLASHES), 0, 0);
-
-      return $this->pollGeolocationAnswer($lastPosition, $errorMessage);
-   }
-
-   /**
-    * Polls in the DB for a new geolocation entry with ID higher than the given one
-    * Timeouts if no new entry after a few seconds
-    * @param array $lastPosition
-    * @param string $errorMessage the error message to return to the caller
-    * @return boolean true if a new position found before timeout
-    */
-   protected function pollGeolocationAnswer($lastPosition, &$errorMessage) {
```

Comment:
If you drop the polling system (and I would agree with that) then the endpoint will behave differently. 

With the polling, the HTTP response code would be 200 only if a new position was recorded in DB.

Without the polling HTTP 200 will be send when the backend sucessfully send the geolocation request only.

If this PR is accepted, then we may need to update some documentation in the gh pages branch.